charts/dmp-roadmap/README.md

```diff
@@ -83,13 +83,13 @@ Helm chart for DMP Roadmap - A Data Management Planning tool that helps research
 | assets.tolerations | list | `[]` | Tolerations for the assets deployment |
 | assets.volumeMounts | list | `[]` | Additional volume mounts for the assets deployment |
 | assets.volumes | list | `[]` | Additional volumes for the assets deployment |
```

Comment:
These changes are generator differences, based on the values.yaml being updated.